DOItools.php

```diff
@@ -568,7 +578,7 @@ function checkTextForMetas($text){
     $authors = formatAuthors($author_list, true);
     foreach ($authors as $no => $auth) {
       $names = explode (', ', $auth);
-      $newp["author" . ($no + 1)] = $names[0];
```

Comment:
last and author are aliases, so it should be fine.  People prefer last when there is a first, and it is split in this case.